boards/arm/cc3220sf_launchxl/doc/index.rst

```diff
@@ -152,17 +152,28 @@ Prerequisites:
 
    Remove power from the board (disconnect USB cable) before switching jumpers.
 
-#. Install TI OpenOCD
+#. Install OpenOCD
 
-   Clone the TI OpenOCD git repository from: http://git.ti.com/sdo-emu/openocd.
-   Follow the instructions in the Release Notes in that repository to build
-   and install.
+   You can obtain OpenOCD by installing the latest Zephyr SDK:
+   https://docs.zephyrproject.org/latest/getting_started/installation_linux.html#zephyr-sdk
```

Comment:
Instead of a hard URL, change to:
```
You can obtain OpenOCD by following these
:ref:`installing the latest Zephyr SDK instructions <zephyr_sdk>`.
```